recipes/android-ndk/all/conanfile.py

```diff
@@ -322,47 +309,51 @@ def package_info(self):
         libcxx_str = str(self.settings_target.compiler.libcxx)
         self.buildenv_info.define("ANDROID_STL", libcxx_str if libcxx_str.startswith("c++_") else "c++_shared")
 
-
         # TODO: conan v1 stuff to remove later
-        self.env_info.PATH.append(self.package_folder)
-        self.env_info.ANDROID_NDK_ROOT = self.package_folder
-        self.env_info.ANDROID_NDK_HOME = self.package_folder
-        cmake_system_processor = self._cmake_system_processor
-        if cmake_system_processor:
-            self.env_info.CONAN_CMAKE_SYSTEM_PROCESSOR = cmake_system_processor
-        else:
-            self.output.warn("Could not find a valid CMAKE_SYSTEM_PROCESSOR variable, supported by CMake")
-        self.env_info.NDK_ROOT = self._ndk_root
-        self.env_info.CHOST = self._llvm_triplet
-        self.env_info.CONAN_CMAKE_FIND_ROOT_PATH = ndk_sysroot
-        self.env_info.SYSROOT = ndk_sysroot
-        self.env_info.ANDROID_NATIVE_API_LEVEL = str(self.settings_target.os.api_level)
-        self.env_info.CONAN_CMAKE_TOOLCHAIN_FILE = toolchain
-        self.env_info.CC = self._define_tool_var("CC", "clang")
-        self.env_info.CXX = self._define_tool_var("CXX", "clang++")
-        self.env_info.AR = self._define_tool_var("AR", "ar", bare)
-        self.env_info.AS = self._define_tool_var("AS", "as", bare)
-        self.env_info.RANLIB = self._define_tool_var("RANLIB", "ranlib", bare)
-        self.env_info.STRIP = self._define_tool_var("STRIP", "strip", bare)
-        self.env_info.ADDR2LINE = self._define_tool_var("ADDR2LINE", "addr2line", bare)
-        self.env_info.NM = self._define_tool_var("NM", "nm", bare)
-        self.env_info.OBJCOPY = self._define_tool_var("OBJCOPY", "objcopy", bare)
-        self.env_info.OBJDUMP = self._define_tool_var("OBJDUMP", "objdump", bare)
-        self.env_info.READELF = self._define_tool_var("READELF", "readelf", bare)
-        if self._ndk_version_major < 23:
-            self.env_info.ELFEDIT = self._define_tool_var("ELFEDIT", "elfedit")
-        if self._ndk_version_major >= 22:
-            self.env_info.LD = self._define_tool_var_naked("LD", "ld")
-        else:
-            self.env_info.LD = self._define_tool_var("LD", "ld")
-        self.env_info.ANDROID_PLATFORM = f"android-{self.settings_target.os.api_level}"
-        self.env_info.ANDROID_TOOLCHAIN = "clang"
-        self.env_info.ANDROID_ABI = self._android_abi
-        self.env_info.ANDROID_STL = libcxx_str if libcxx_str.startswith("c++_") else "c++_shared"
-        self.env_info.CMAKE_FIND_ROOT_PATH_MODE_PROGRAM = "BOTH"
-        self.env_info.CMAKE_FIND_ROOT_PATH_MODE_LIBRARY = "BOTH"
-        self.env_info.CMAKE_FIND_ROOT_PATH_MODE_INCLUDE = "BOTH"
-        self.env_info.CMAKE_FIND_ROOT_PATH_MODE_PACKAGE = "BOTH"
+        if Version(conan_version).major < 2:
```

Comment:
This is not used in conan center, should be safe to drop all of it

https://conan.io/center/android-ndk?tab=usedby

Reply:
What do you want to drop exactly? There are still users of `android-ndk` who expect to cross-build recipes relying on conan v1 build helpers. The fact that no other CCI recipes depend on `android-ndk` is unrelated.

Reply:
Like everything they should know to use the latest features to stay with CCI. Or use revisions.

We've always been on the bleeding edge and that's not about to change.

They should update.

Reply:
What? You don't understand. What you suggest would break cross-build to Android of all CCI recipes not relying on conan v2 build helpers.

Reply:
Perfect. Let them upgrade 🚀 

Or, the docs have solutions.

https://github.com/conan-io/conan-center-index#how-to-consume-recipes
https://github.com/conan-io/conan-center-index/blob/master/docs/consuming_recipes.md